src/shogun/classifier/svm/SVM.h

```diff
@@ -64,6 +64,8 @@ class SVM : public KernelMachine
 		 */
 		SVM(float64_t C, std::shared_ptr<Kernel> k, std::shared_ptr<Labels> lab);
 
+
```

Comment:
extra space, I think you meant to put the space between constructor and destructor